docs/README.md

```diff
@@ -83,41 +83,57 @@ The Notebooks can run on various compute targets; the ones currently supported o
 
 # Productionalization
 
-Designing and building a production data pipeline for a Predictive Maintenance solution can be a relatively non-trivial task. One of the main reasons being that feature engineering (featurization) requires not only new, but also historic data. For reasons outlined below, this solution was built to support real-time featurization and scoring.
+As it is often the case when dealing with time series, feature engineering (featurization) requires access to historical data in addition to the new data. For that reason, designing and building a production data pipeline for a Predictive Maintenance solution can be a relatively non-trivial task.
+
+Generally, predictions can be made either in real time or on a batch basis. This solution was built to support near real-time featurization and scoring, which means up-to-date predictions are generated almost as soon as new data is available.
```

Comment:
Why are we keeping the weasel words?
The "near" part of near-real-time is an effect of latency inherent to all computing.  There is no such thing as a truly real-time system even if only due to propagation delay between sensor and microprocessor.
Further, in the context of generating a prediction using an ML model, the term itself "real-time" is meant to indicate that a stream of data is being scored.
Please remove "near" and remove "almost".